incubation.md

```diff
@@ -0,0 +1,79 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which new Jupyter Subprojects are created and evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a GitHub repository with a development team that follows the Jupyter governance and contribution model. The goals of the incubation process are:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while complying with corporate contribution restrictions.
```

Comment:
Is an org of explicitly unofficial projects more visible than mailing-list announcements?